clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerServiceDelegatorTest.java

```diff
@@ -544,7 +544,7 @@ public void testKafkaConsumerServiceResubscriptionConcurrency() throws Exception
     KafkaConsumerServiceDelegator delegator =
         new KafkaConsumerServiceDelegator(mockConfig, consumerServiceBuilder, isAAWCStoreFunc);
     PubSubTopicPartition realTimeTopicPartition =
-        new PubSubTopicPartitionImpl(TOPIC_REPOSITORY.getTopic(Version.composeRealTimeTopic(storeName)), 0);
+        new PubSubTopicPartitionImpl(TOPIC_REPOSITORY.getTopic(storeName + Version.REAL_TIME_TOPIC_SUFFIX), 0);
```

Comment:
Yea, I actually wanted to reduce the use of that method; and eventually remove it. Let me mark it @Deprecated 
Another reason for not using it, is that it does not need to be Version.composeRealTimeTopic(storeName) in tests like this, it can just be any random string.